backend/organization/views/project_views.py

```diff
@@ -83,7 +87,10 @@ def get_queryset(self):
         if 'skills' in self.request.query_params:
             skill_names = self.request.query_params.get('skills').split(',')
             skills = Skill.objects.filter(name__in=skill_names)
-            projects = projects.filter(skills__in=skills).distinct('id')
+            # We use distinct to deduplicate selected rows. We
+            # must use order_by in conjunction with distinct:
+            # https://docs.djangoproject.com/en/dev/ref/models/querysets/#django.db.models.query.QuerySet.distinct
+            projects = projects.filter(skills__in=skills).order_by('id').distinct('id')
```

Comment:
Wanted to confirm that ordering by ID is desired. I think another option could be 

```suggestion
            projects = projects.filter(skills__in=skills).order_by('name').distinct('name')
```

Reply:
Yeah order by id is perfect. We do not have custom primary keys anywhere.

Reply:
Sounds good; I'll explore this a bit. 

Note for me: I need to learn how to more easily create some test data with properties like rating and what not.

Reply:
Yeah, we definitely still need to build on the test data script. There's still some information missing in there such as images, rating, ... I'll create an issue for that!
Quick note on how the project order on the main page works:
Projects are ordered by rating. A new project starts with the default value of 100, which is the highest possible value. 
When a project has been on top for ~1 week we manually adapt its rating based on the stats it produced and based on how interesting we believe it will be for new members. We also try to have projects from different categories at the top. We will automize this process more and more but with the current number of projects, this is still the best way to do it in terms of time invested and quality.